mastodon/Mastodon.py

```diff
@@ -1190,7 +1190,7 @@ def __api_request(self, method, endpoint, params={}, files={}, do_ratelimiting=T
                     raise MastodonAPIError('Endpoint not found.')
 
                 if isinstance(response, dict) and 'error' in response:
-                    raise MastodonAPIError("Mastodon API returned error: " + str(response['error']))
+                    raise MastodonAPIError("Mastodon API returned error: " + response['error'].encode('utf-8').strip())
```

Comment:
This won't work on python 3

```
>>> "Mastodon API returned error: " + u"blahblah".encode('utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: must be str, not bytes
```

Reply:
```
media_dict = mastodon.media_post("output.jpg")
status = ""
status.encode('utf-8').strip()
mastodon.status_post(status, in_reply_to_id=None, media_ids=[media_dict])
```

I use `python3.5 monscript.py `  
If status is empty, api send an error (empty text) and api return backtrace send an error : `unicodeencodeerror 'ascii' codec can't encode character u' xe9'`